### PR TITLE
Rename mask to masked, use LoadXxx with path based constructors, make fonts work as urls.

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: convert/stl/fromStl.js [hunk 1]
```diff
@@ -13,10 +13,7 @@ const toParser = (format) => {
   }
 };
 
-export const fromStl = async (
-  stl,
-  { format = 'ascii', geometry = 'graph' } = {}
-) => {
+export const fromStl = async (stl, { format = 'ascii' } = {}) => {
```
Comment (BarbourSmith) [inline convert/stl/fromStl.js:16]: What if it's binary?

File: nb/api/Spiral.nb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+md`
+[index](#https://raw.githubusercontent.com/jsxcad/JSxCAD/master/nb/api/index.nb)
+### Spiral()
+Parameter|Default|Type
+---|---|---
+op|Point|Function or shape to build from.
```
Comment (BarbourSmith) [inline nb/api/Spiral.nb:6]: The documentation on these is really nice :)

**Review by BarbourSmith — APPROVED** [review]
After a refresh I'm still not seeing some images, but what I can see looks great 👍 👍

File: convert/stl/fromStl.js (hunk 1 above)
Comment (pentacular) [reply convert/stl/fromStl.js:16]: This is the default, binary can be passed in.
Probably it should check that it's one of the accepted formats.

File: nb/api/Spiral.nb (hunk 2 above)
Comment (pentacular) [reply nb/api/Spiral.nb:6]: Yeah, moving it to the table form helped a lot, since they're mostly not positional arguments now.

**Pull request merged**